src/screens/Home/index.js

```diff
@@ -0,0 +1,5 @@
+function Home() {
+  return null;
```

Comment:
add a redirect here to `/login` ?
when you have session just check if there is a session and redirect, OR delete this screen

Reply:
is a partial feature, the card asks to add to the empty screen